annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastFieldResolver.java

```diff
@@ -27,16 +27,24 @@ private boolean isOverriddenIn(FastTypeElement fastType, FastFieldElement fastFi
     }
 
     private List<FastFieldElement> getFastFieldsFor(Element aClass) {
-        return cache.computeIfAbsent(aClass,
+        List<FastFieldElement> fastFieldElements = cache.computeIfAbsent(aClass,
             key -> key.getEnclosedElements().stream()
-            .filter(element -> element.getAnnotation(FastField.class) != null)
-            .map(element -> new FastFieldElement((ExecutableElement) element))
-            .collect(Collectors.toList()));
+                .filter(element -> element.getAnnotation(FastField.class) != null)
+                .map(element -> new FastFieldElement((ExecutableElement) element))
+                .collect(Collectors.toList()));
+        if (containsInvalidMethodParameters(fastFieldElements)) {
```

Comment:
We need to collect all annotated fields here. But the code which can be executed between this point and validation should be aware that some `FastFieldElements` can be invalid.